test/package.test.js

```diff
@@ -0,0 +1,21 @@
+import { describe, expect, it } from 'vitest'
+import packageJson from '../package.json' with { type: 'json' }
+
+describe('package.json', () => {
+  it('should have the correct name', () => {
+    expect(packageJson.name).toBe('@hyparam/monorepo')
+  })
+  it('should have a valid version', () => {
+    expect(packageJson.version).toMatch(/^\d+\.\d+\.\d+$/)
+  })
+  it('should have MIT license', () => {
+    expect(packageJson.license).toBe('MIT')
+  })
+  it('should have precise dependency versions', () => {
```

Comment:
How did this test pass when there were non-precise devDependencies?

Reply:
exactly, good question. Looking at it!

Reply:
I forgot to create the scripts + the github action for this. Done at https://github.com/hyparam/hyperparam-cli/pull/130/commits/3b9ba5a3b13a99781a0bd709a8dcdc5d27a3dd4d

Reply:
Finally fixed: the test fails: https://github.com/hyparam/hyperparam-cli/actions/runs/12323300110/job/34398644476?pr=130